docs/GettingStarted.md

```diff
@@ -0,0 +1,244 @@
+So, instead we have chosen to use a Python `tuple` as equivalent to a Matlab `cell array`
+(a Python `list` which contains non-numeric elements will also be interpreted as a Matlab `cell array`).
+
+Thus the following code in Matlab:
+
+```matlab
+proj = struct('u', [1 1 0], 'v', [-1 1 0], 'type', 'rrr');
+ws = cut_sqw('file.sqw', proj, [-2, 0.05, 2], [-0.1, 0.1], [-0.1, 0.1], [4, 6]);
+tri = sw_model('triAF', 1);
+fwhm = 0.75;
+scalefactor = 1;
+kk = multifit_sqw(ws);
+kk = kk.set_fun(@tri.horace_sqw);
+kk = kk.set_pin({[J fwhm scalefactor], 'mat', {'J_1'}, 'hermit', false, 'formfact', true, 'resfun', 'gauss'});
+tic
+ws_sim = kk.simulate()
+toc
+```
+
+would be in Python:
+
+```python
+proj = {'u':[1, 1, 0], 'v':[-1, 1, 0], 'type':'rrr'}
+ws = m.cut_sqw('file.sqw', proj, [-2, 0.05, 2], [-0.1, 0.1], [-0.1, 0.1], [4, 6])
+tri = m.sw_model('triAF', 1)
+fwhm = 0.75
+scalefactor = 1
+kk = m.multifit_sqw(ws)
+kk = kk.set_fun(tri.horace_sqw)
+kk = kk.set_pin(([J, fwhm, scalefactor], 'mat', ['J_1'], 'hermit', false, 'formfact', true, 'resfun', 'gauss'))
+m.tic()
+ws_sim = kk.simulate()
+m.toc()
+```
+
+### Setup
+
+In order to get started with pace-neutrons, first begin a python session either in your terminal
+by running either the `python` or `python3` command or by creating a `.py` file. Once this is done, 
+the following commands must be run in order for Horace commands to be useable:
+
+```python
+from pace_neutrons import Matlab
+m = Matlab()
+```
+
+**Note:** if you assign `Matlab()` to a different variable name this would be the new prefix to 
+the Horace matlab commands.
+
+### Example commands
+
+#### Example
+
+Using Horace:
+```matlab
+proj = projaxes([-0.5, 1, 0], [0, 0, 1], 'type', 'rrr');
+w1 = cut_sqw('ei30_10K.sqw', proj, [0.1, 0.02, 0.5], [1.5, 2.5], [0.4, 0.5], [3, 0.5, 20]);
+hf = plot(w1);
+```
+
+Using pace-neutrons:
+```python
+from pace_neutrons import Matlab
+m = Matlab()
+proj = m.projaxes([-0.5, 1, 0], [0, 0, 1], 'type', 'rrr')
+w1 = m.cut_sqw('ei30_10K.sqw', proj, [0.1, 0.02, 0.5], [1.5, 2.5], [0.4, 0.5], [3, 0.5, 20])
+hf = m.plot(w1)
+```
+
+**Note:** The above example will not work for you due the absence of the `'ei30_10K.sqw'` file.
+
+Additional examples can be found in the demo scripts detailed in the next section.
+The examples found in the demo are complete with the required data files ready for use.
+
+#### Demo
+
+A demo for pace-neutrons is found [here](https://github.com/pace-neutrons/pace-python-demo). This can be 
+downloaded by first navigating to the provided github repository and clicking on the green code button and
+selecting the `Download ZIP` option. This demo is complete with a demo `.py` script as well as a `.ipynb` 
+file for use in a Jupyter notebook.
+
+
+#### Horace GUI
+
+The Horace GUI can be opened, following the setup procedure outlined above, by 
+running `m.horace()`
+
+## Jupyter notebook specifics
```

Comment:
Link to other doc on magic

Reply:
How do you link to another doc? I don't know what the url will be to the thing once it is merged

Reply:
@oerc0122